pkg/services/featuremgmt/registry.go

```diff
@@ -288,7 +288,7 @@ var (
 			Name:            "nestedFolders",
 			Description:     "Enable folder nesting",
 			State:           FeatureStateAlpha,
```

Comment:
question: should we also update the feature toggle to `beta` state? [Docs](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/#alpha-feature-toggles) make it sound like `alpha` features aren't ready to be used in cloud yet, but we probably will want this to be tested in cloud soon.

Reply:
Sure!